src/lib/formatter/RuleResultRecombinator.ts

```diff
@@ -110,7 +114,12 @@ export class RuleResultRecombinator {
 				const escapedUrl = this.safeHtmlEscape(v.url);
 
 				problemCount++;
-				violations += `<violation severity="${v.severity}" line="${v.line}" column="${v.column}" endLine="${v.endLine}" endColumn="${v.endColumn}" rule="${escapedRuleName}" category="${escapedCategory}" url="${escapedUrl}">${escapedMessage}</violation>`;
+				if (normalizeSeverity) {
+					violations += `<violation severity="${v.severity}" normalizedSeverity="${v.normalizedSeverity}" line="${v.line}" column="${v.column}" endLine="${v.endLine}" endColumn="${v.endColumn}" rule="${escapedRuleName}" category="${escapedCategory}" url="${escapedUrl}">${escapedMessage}</violation>`;
```

Comment:
Could the `severity="${v.severity}" normalizedSeverity="${v.normalizedSeverity}"` portions be broken out and placed into their own string? This would avoid the possibility of the strings getting out of sync. 

```Javascript
let severityAttributes = `severity="${v.severity}"`;
if (normalizeSeverity) {
    severityAttributes += ` normalizedSeverity="${v.normalizedSeverity}"`;
}


violations += `<violation ${severityAttributes} line="${v.line}" column="${v.column}" endLine="${v.endLine}" endColumn="${v.endColumn}" rule="${escapedRuleName}" category="${escapedCategory}" url="${escapedUrl}">${escapedMessage}</violation>`;
```
```